### PR TITLE
fix: Delay shutdown in graceful mode.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

**Comment by bgrozev** [comment]
@JonathanLennox can you review again please?

File: jvb/src/main/resources/reference.conf [hunk 1]
```diff
@@ -321,8 +321,11 @@ videobridge {
   }
 
   version {
-    // Wheather to announe the jitsi-videobridge version to clients in the ServerHello message.
+    # Wheather to announe the jitsi-videobridge version to clients in the ServerHello message.
```
Comment (JonathanLennox) [inline jvb/src/main/resources/reference.conf:324]: Typos: "Whether", "announce".  This is old code but might as well fix it when you're editing the line?

**Review by JonathanLennox — DISMISSED** [review]

File: jvb/src/main/java/org/jitsi/videobridge/Videobridge.java [hunk 2]
```diff
@@ -255,18 +264,23 @@ public Videobridge(
...
      */
     private void enableGracefulShutdownMode()
     {
-        if (!shutdownInProgress)
+        if (shutdownRequestedTime == null)
         {
             logger.info("Entered graceful shutdown mode");
         }
-        this.shutdownInProgress = true;
+        this.shutdownRequestedTime = clock.instant();
```
Comment (JonathanLennox) [inline jvb/src/main/java/org/jitsi/videobridge/Videobridge.java:275]: Hm, just noticed this - if enableGracefulShutdownMode is called more than once, this reset the shutdown time.  Is that what we want?

File: jvb/src/main/java/org/jitsi/videobridge/Videobridge.java [hunk 3]
```diff
@@ -598,7 +630,7 @@ public void stop()
     public OrderedJsonObject getDebugState(String conferenceId, String endpointId, boolean full)
     {
         OrderedJsonObject debugState = new OrderedJsonObject();
-        debugState.put("shutdownInProgress", shutdownInProgress);
+        debugState.put("shutdownInProgress", (shutdownRequestedTime != null));
```
Comment (JonathanLennox) [inline jvb/src/main/java/org/jitsi/videobridge/Videobridge.java:633]: Do we want to have the actual requested time in the debug state?

Comment (bgrozev) [reply jvb/src/main/java/org/jitsi/videobridge/Videobridge.java:633]: I think having it in the logs is sufficient.

**Review by JonathanLennox — APPROVED** [review]

**Pull request merged**